File: python/Hardware.py
from Product import Product

class Hardware(Product):
	# atribut private
	brand = ""
	model = ""

	# getter/setter
	def setbrand(self, brand):
		self.brand = brand

	def getbrand(self):
		return self.brand

	def setmodel(self, model):
		self.model = model

	def getmodel(self):
		return self.model
	

File: php/Hardware.php
<?php

include "Product.php";

class Hardware extends Product{
	// atribut private
    private $brand;
    private $model;

	// constructor
	public function __construct($brand = "", $model = ""){
        $this->brand = $brand;
        $this->model = $model;
	}

	// setter/getter
    public function setbrand($brand){
		$this->brand = $brand;
	}
    public function getbrand(){
		return $this->brand;
	}

    public function setmodel($model){
		$this->model = $model;
	}
    public function getmodel(){
		return $this->model;
	}
}

?>